WebCryptoAPI/idlharness.https.any.js

```diff
@@ -0,0 +1,23 @@
+// META: script=/resources/WebIDLParser.js
+// META: script=/resources/idlharness.js
+setup(function() {
+    var request = new XMLHttpRequest();
+    request.open("GET", "../interfaces/WebCryptoAPI.idl");
```

Comment:
If there's a typo here, the test will just time out, because there's no such thing as async setup. Can you copy the style of encoding/idlharness.any.js instead? Thanks to @lukebjerring that style is used in lots of idlharness.js tests now, and it makes them quite a bit more compat/uniform.

Reply:
done